src/css/jwplayer.less

```diff
@@ -0,0 +1,261 @@
+    pointer-events: all;
+}
+
+.jw-skip {
+    position: absolute;
+    right: 1em;
+    display: inline-block;
+    float: right;
+    width: 7em;
+    height: 2em;
+}
+
+.jw-skip-image {
+    position: relative;
+    display: none;
+}
+
+.jw-captions {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    cursor: pointer;
+}
+
+.jw-logo {
+    position: absolute;
+    top: 8px;
+    right: 8px;
+    margin-bottom: 0;
+    cursor: pointer;
+}
+
+/***** Start Right Click *****/
+.jw-click {
+    position: absolute !important;
+    z-index: 999 !important;
+    width: 320px !important;
+    height: auto !important;
+    font-family: 'MS Sans Serif', Geneva, sans-serif !important;
+    font-size: 10px !important;
+    background-color: rgb(255, 255, 255) !important;
+    border: 1px solid rgb(188, 188, 188) !important;
+    border-radius: 5px !important;
+    -webkit-box-shadow: rgba(0, 0, 0, 0.1) 5px 5px 7px rgba(255, 255, 255, 0.3) 0 1px 0 inset !important;
+    box-shadow: rgba(0, 0, 0, 0.1) 5px 5px 7px rgba(255, 255, 255, 0.3) 0 1px 0 inset !important;
+}
+
+.jw-click div {
+    padding: 8px 21px !important;
+    margin: 0 !important;
+    font-family: 'MS Sans Serif', Geneva, sans-serif !important;
+    font-size: 10px !important;
+    color: inherit !important;
+    background-color: rgb(255, 255, 255) !important;
+    border: none !important;
+}
+
+.jw-click-item {
+    padding: 8px 21px !important;
+    text-align: left !important;
+    cursor: pointer !important;
+}
+
+.jw-click-item:hover {
+    color: rgb(255, 255, 255) !important;
+    background-color: rgb(89, 89, 89) !important;
+}
+
+.jw-click-item a {
+    color: rgb(0, 0, 0) !important;
+    text-decoration: none !important;
+}
+
+.jw-click hr {
+    width: 100% !important;
+    padding: 0 !important;
+    margin: 0 !important;
+    border: 1px solid rgb(233, 233, 233) !important;
+}
+
+/**** END RIGHT CLICK ****/
+
+.jwplayer {
+    position: relative;
+    display: block;
+    min-height: 0;
+    opacity: 0;
+    -webkit-transition: opacity .25s ease;
+    transition: opacity .25s ease;
+}
+
+.jw-main {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    -webkit-transition: opacity .25s ease;
+    transition: opacity .25s ease;
+}
+
+.jw-video, .jw-controls {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    -webkit-transition: opacity .25s ease;
+    transition: opacity .25s ease;
+}
+
+.jw-video {
+    overflow: hidden;
+    opacity: 0;
+    visibility: hidden;
+}
+
+.jw-video video {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    margin: auto;
+    background: transparent;
+}
+
+.jw-instream {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    display: none;
+}
+
+.jw-aspect {
+    display: none;
+}
+
+.jwplayer.aspect-mode {
```

Comment:
Everywhere else, the classes are prefixed with jw-, we should continue that here.